api/services/advanced_prompt_template_service.py

```
@@ -13,7 +13,7 @@
     CONTEXT,
 )
 from models.model import AppMode
-
+from account_service import *
```

Ruff (0.8.0)

`from account_service import *` used; unable to detect undefined names

(F403)